molgenis-data/src/main/java/org/molgenis/data/DataConverter.java

```diff
@@ -20,9 +19,6 @@
 import org.springframework.core.convert.ConverterNotFoundException;
 import org.springframework.core.convert.support.DefaultConversionService;
 
-@SuppressFBWarnings(
-    value = "NP_BOOLEAN_RETURN_NULL",
-    justification = "We want to return Boolean.TRUE, Boolean.FALSE or null")
```

Comment:
@fdlk What does Sonar say about this one?

Reply:
Nothing, I think because we suppress the sonar equivalent